libs/newt/Makefile

```diff
@@ -24,6 +24,7 @@ PKG_CPE_ID:=cpe:/a:fedorahosted:newt
 
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
+PYTHON3_PKG_BUILD:=0
```

Comment:
fwiw: i liked the old mechanism to just define a `Build/Compile` rule for python packages; then if any package wants it differently, they just re-define [or clear]  `Build/Compile`;

no major objections for this new one;
the old one, felt a bit more OpenWrt-ish on how packages are built;
well, as OpenWrt-ish as Python package buils go;

Reply:
python3-package.mk is used by more than just Python packages now: some packages only want to use the `PYTHON3_*` variables (like [libmraa](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/libs/libmraa/Makefile), [samba4](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/net/samba4/Makefile), and [openvswitch](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/net/openvswitch/Makefile)), some packages build a main library/binary but also have a Python bindings/helper package that can use `Py3Package` (like [libsearpc](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/libs/libsearpc/Makefile) and [nmap](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/net/nmap/Makefile)).

Most of these define their own `Build/Compile`, but I doubt many of these package authors knew they had to define it to override the one defined in python3-package.mk. (Python package authors like us are probably the only ones that know the Python build process well enough to know this.)

For newt, I had to define `Build/Compile` [back to the default](https://github.com/openwrt/packages/blob/e291d680558f40ae6d0d0a7006ac1991a6910d2f/libs/newt/Makefile#L96) just to override the Python version.

I think it would be good to have a variable that packages can set to opt-out of the default Python build recipe and do something custom. In the future, python3-package.mk may override more than just `Build/Compile`.

(An alternative would be to have two include files, one for Python packages and one for non-Python packages, but this seems more complicated than it needs to be for now.)

I would have preferred `PYTHON3_PKG_BUILD` to be opt-in (like `PKG_INSTALL:=1`) but because it is mainly used by Python packages, it is opt-out (`PYTHON3_PKG_BUILD:=0` to not have the default build recipe). Python packages will not need to have this set.

Reply:
> I would have preferred `PYTHON3_PKG_BUILD` to be opt-in (like `PKG_INSTALL:=1`) but because it is mainly used by Python packages, it is opt-out (`PYTHON3_PKG_BUILD:=0` to not have the default build recipe). Python packages will not need to have this set.

yeah; opt-in would have been nice, but it's a lot of work;
fine to do it like this